packages/fx-core/resource/package.nls.json

```diff
@@ -295,7 +295,7 @@
   "core.createProjectQuestion.apiSpecInputUrl.label": "Enter OpenAPI Description Document Location",
   "core.createProjectQuestion.OpenAIPluginDomain": "OpenAI Plugin Manifest",
   "core.createProjectQuestion.OpenAIPluginDomain.placeholder": "Enter your website domain or manifest URL",
-  "core.createProjectQuestion.invalidUrl.message": "Please enter a valid URL",
+  "core.createProjectQuestion.invalidUrl.message": "Please enter a valid HTTP URL without authentication to access your OpenAPI description document.",
```

Comment:
typically in message we don't add "Please", so can remove it.

description document -> specification?

Reply:
According to [Bug 25257315](https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/25257315): [API ME] Update "OpenAPI Specfication" to "OpenAPI Description Document", OpenAPI description document is what we should use.  “OpenAPI Specification is a standard, OpenAPI Description document is to describe an API that conforms to that standard“